akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -284,6 +284,7 @@
                         <execution>
                             <goals>
                                 <goal>integration-test</goal>
+                                <goal>verify</goal>
```

Comment:
true that the build is not failing, but I tried with `mvn integration-test` and it's the same thing with that even though it was included here